demisto_sdk/commands/download/downloader.py

```diff
@@ -254,7 +254,7 @@ def handle_layout(self, layout_string, scripts_mapper):
         file_json_object = json.loads(layout_string)
         layout_name = file_json_object.get('name')
         if layout_name and (layout_name in self.input_files or self.all_custom_content):
-            for tab in file_json_object.get('detailsV2', {}).get('tabs', ()):
+            for tab in (file_json_object.get('detailsV2') or {}).get('tabs', ()):
```

Comment:
I replaced detailsV2 with indicatorsQuickView and it downloaded the indicator layout.

Reply:
```
    def handle_layout(self, layout_string, scripts_mapper):
        # In case the layout uses a custom script, replace the id value of the script with its name
        file_json_object = json.loads(layout_string)
        layout_name = file_json_object.get('name')
        if layout_name and (layout_name in self.input_files or self.all_custom_content):
            json_objects_with_tabs = []
            for obj in file_json_object:
                if isinstance(obj, dict) and obj.get('tabs', ''):
                    json_objects_with_tabs.append(obj)
                    
            for obj in json_objects_with_tabs:
                for tab in file_json_object.get(obj, {}).get('tabs', ()):
                    for section in tab.get('sections', ()):
                        for item in section.get('items', ()):
                            script_id = item.get('scriptId')
                            if script_id and script_id in scripts_mapper:
                                layout_string = layout_string.replace(script_id, scripts_mapper[script_id])

        return layout_string
```
Does it make sense to go through all the objects that contains tabs?

Reply:
```
    def handle_layout(self, layout_string, scripts_mapper):
        # In case the layout uses a custom script, replace the id value of the script with its name
        file_json_object = json.loads(layout_string)
        layout_name = file_json_object.get('name')
        if layout_name and (layout_name in self.input_files or self.all_custom_content):

            json_objects_with_tabs = []
            for obj in file_json_object:
                if isinstance(file_json_object[obj], dict):
                    if file_json_object[obj].get('tabs', None):
                        json_objects_with_tabs.append(obj)
                    
            for obj in json_objects_with_tabs:
                for tab in file_json_object.get(obj, {}).get('tabs', ()):
                    for section in tab.get('sections', ()):
                        for item in section.get('items', ()):
                            script_id = item.get('scriptId')
                            if script_id and script_id in scripts_mapper:
                                layout_string = layout_string.replace(script_id, scripts_mapper[script_id])

        return layout_string

```

Does it make sense to go through all the json objects that contain a tab?
EDIT: modified the code a bit to get it working.

Reply:
What do you mean? Why change `detailsV2`to `indicatorsQuickView`?

Reply:
if there is a script associated to a layout this function seems to rename the scriptId of it. Not only detailsV2 but also indicatorsQuickView and indicatorsDetails hold the same type of content in the yml but for indicator layouts. That's just my observation. so if they are practically the same, the function should also look into these objects for scriptIds.

for the first comment, I was trying to download an indicator layout and it failed on detailsV2 but it looked like this function was actually looking for indicatorsQuickView.

Reply:
I see, thanks for the explanation. This looks valid but requires proper unit tests to merge, we'll do it.
I will merge this fix for now anyway, to allow downloading cases with a `None` value until `indicatorsQuickView` is handled.